docker-compose.yml

```diff
@@ -1,9 +1,8 @@
 services:
-  app:
+  observer:
     build:
       context: .
       dockerfile: Dockerfile
-    container_name: gnosh-observer
```

Comment:
`container_name` isn't really necessary and can even be counterproductive in some cases (esp. when scaling services).